.gitlab/pipeline/zombienet/substrate.yml

```diff
@@ -50,23 +57,29 @@ zombienet-substrate-0000-block-building:
 
 zombienet-substrate-0001-basic-warp-sync:
   extends:
-    - .zombienet-substrate-common
+    - .zombienet-substrate-warp-sync-common
   script:
     - /home/nonroot/zombie-net/scripts/ci/run-test-local-env-manager.sh
       --local-dir="${LOCAL_DIR}/0001-basic-warp-sync"
       --test="test-warp-sync.zndsl"
 
 zombienet-substrate-0002-validators-warp-sync:
   extends:
-    - .zombienet-substrate-common
+    - .zombienet-substrate-warp-sync-common
+  before_script:
+    - !reference [.zombienet-substrate-warp-sync-common, before_script]
+    - cp ${LOCAL_DIR}/0001-basic-warp-sync/chain-spec.json ${LOCAL_DIR}/0002-validators-warp-sync
```

Comment:
Is `cp` a leftover now?

Reply:
Yes, sorry, removed

Reply:
Actually this is needed : https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/4887696 .

Reply:
Hm, could you explain why? We have a link there, right?

Reply:
I guess it's because when we do `/home/nonroot/zombie-net/scripts/ci/run-test-local-env-manager.sh --local-dir="${LOCAL_DIR}/0001-basic-warp-sync"` only the `${LOCAL_DIR}/0002-validators-warp-sync` is made available to the test somehow (uploaded to the container or something). So even if the link is there, the `${LOCAL_DIR}/0001-basic-warp-sync` folder is not present

Reply:
makes sense. Thank you.